pkg/resources/grant_privileges_to_role.go

```diff
@@ -442,12 +443,15 @@ func (v GrantPrivilegesToAccountRoleID) String() string {
 }
 
 func CreateGrantPrivilegesToRole(d *schema.ResourceData, meta interface{}) error {
+	logging.DebugLogger.Printf("[DEBUG] Entering create grant privileges to role")
```

Comment:
since you already have a debug logger, is it necessary to prefix each line with [DEBUG]?